### PR TITLE
phpBB 3.3 compatibility

### PR DESCRIPTION
**Pull request by rxu**

### CODE REVIEW

**Comment by Galixte** [comment]
Hi @lavigor,

could you merge this, please?

Thanks to @rxu.

Regards.

File: acp/upload_module.php [hunk 1]
```diff
@@ -59,20 +59,20 @@ function main($id, $mode)
 		$phpbb_link_template = '#^(https://)www.phpbb.com/customise/db/download/([0-9]*?)(/composer|/manual)?/?(\?sid\=[a-zA-Z0-9]*?)?$#i';
 
 		// Work with objects class instead of $this.
-		objects::$cache = &$cache;
```
Comment (lavigor) [inline acp/upload_module.php]: @rxu could you please comment the necessity for this deletion?
Tested on phpBB 3.3.0 w/o this change, no errors.

Comment (rxu) [reply acp/upload_module.php]: Honestly I don't remember already, Probably I had some issues with PHP 7.4 or something but now I failed to reproduce :)

Comment (lavigor) [reply acp/upload_module.php]: OK, then I'll discard this part for now.

**Comment by lavigor** [comment]
Sorry that it took such a long (really long) time to just check and merge this PR.

**Comment by Galixte** [comment]
@lavigor thank you. ;)

**Pull request merged**